features/page_objects/claim_form_page.rb

```diff
@@ -22,7 +22,6 @@ class ClaimFormPage < SitePrism::Page
   element :court, "#s2id_autogen1"
   element :case_type, "#s2id_autogen2"
   element :case_number, "#claim_case_number"
-  # element :claim_advocate_category_junior_alone, "#claim_advocate_category_junior_alone" # TODO: remove if no longer needed
```

Comment:
Can be removed?

Reply:
yep 👍

Reply:
actually has been in later commit but now showing for diff reasons i guess